packages/uui-sort-symbol/lib/uui-sort-symbol.story.ts

```diff
@@ -0,0 +1,65 @@
+import { Story } from '@storybook/web-components';
+import { html } from 'lit-html';
+import '@umbraco-ui/uui-sort-symbol/lib/index';
+
+export default {
+  id: 'uui-sort-symbol',
+  title: 'Symbols/Sort Symbol',
+  component: 'uui-sort-symbol',
+  parameters: {
+    docs: {
+      source: {
+        code: `<uui-sort-symbol></uui-sort-symbol>`,
+      },
+    },
+  },
+  args: {
+    active: false,
+    descending: false,
+  },
+  argTypes: {
+    value: {
+      active: { type: 'boolean' },
+      descending: { type: 'boolean' },
+    },
+  },
+};
+
+export const Overview: Story = props =>
+  html`<style>
+      button {
+        padding: 6px 6px 6px 12px;
+        background-color: transparent;
+        color: inherit;
+        border: none;
+        border-bottom: 1px solid currentColor;
+        cursor: pointer;
+      }
+      uui-sort-symbol {
+        margin-left: 20px;
+      }
+      button:hover {
+        --uui-sort-symbol--hover: 1;
```

Comment:
You use double dashed before hover in the custom property name. I think this is the only place we do that? I think we should remove that for consistency.

Reply:
Good catch, correcting it now :-)